src/Raven.Client/ServerWide/Operations/Configuration/GetServerWideBackupConfigurationOperation.cs

```diff
@@ -34,7 +35,7 @@ public GetServerWideBackupConfigurationCommand(string name)
 
             public override HttpRequestMessage CreateRequest(JsonOperationContext ctx, ServerNode node, out string url)
             {
-                url = $"{node.Url}/admin/configuration/server-wide/backup?name={Uri.EscapeDataString(_name)}";
+                url = $"{node.Url}/admin/configuration/server-wide/tasks?type={OngoingTaskType.Backup}&name={Uri.EscapeDataString(_name)}";
```

Comment:
This breaks the backward compatibility

Reply:
I left the endpoint for backward compatibility